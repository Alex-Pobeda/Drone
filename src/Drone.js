import React from 'react';
import './App.css';

const Drone = () => {
  return (
    <div className="drone" />
  );
};
export default Drone;
