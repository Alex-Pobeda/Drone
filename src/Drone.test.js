import React from 'react';
import ReactDOM from 'react-dom';
import Drone from './Drone';

describe('Drone', () => {
  it('should renders Drone without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Drone/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });
});
