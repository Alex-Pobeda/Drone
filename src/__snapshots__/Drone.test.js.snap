// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drone component Matches the snapshot 1`] = `
<div
  className="drone"
/>
`;
